session-1/assets/css/main.css

```diff
@@ -0,0 +1,35 @@
+.box {
```

Comment:
Use card class to make card component, don't make card component with ustom class CSS live you've made, use bootstrap class. Read this documentation [Card](https://getbootstrap.com/docs/5.1/components/card/)